src/bowlingFrame.js

```diff
@@ -0,0 +1,61 @@
+var BowlingFrame = function(number) {
+
+  this.number = number;
+  this.rollIndex = 0;
+
+  if (number != 1 && game.currentGame[1] === [null,null]) {
+    throw new Error('Your must add to frame one first');
+  };
+
+  if (number !== 1 && game.currentGame[number - 1][0] == null) {
+    throw new Error('You must add to frames in correct order');
+  };
+};
+
+BowlingFrame.prototype.rollOne = function(input) {
+  number = this.number;
+  if (game.currentGame[number][0] != null) {
+    throw new Error('You already have a score here');
+  };
+
+  if (input > 10 || input < 0) {
+    throw new Error('Roll must be between 0 and 10');
+  } else {
+    rollIndex = this.rollIndex;
+    game.currentGame[number][0] = input;
+  };
+
+  game.currentScore();
```

Comment:
The only reason this is working is because your test defines a global variable, `game`. This code wouldn't work in production.